webapp/components/integrations/components/edit_command/index.js

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) 2017 Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import {connect} from 'react-redux';
+import {bindActionCreators} from 'redux';
+import {getCustomTeamCommands, editCommand} from 'mattermost-redux/actions/integrations';
+import {getCommands} from 'mattermost-redux/selectors/entities/integrations';
+
+import EditCommand from './edit_command.jsx';
+
+function mapStateToProps(state, ownProps) {
+    const commandId = ownProps.location.query.id;
+
+    return {
+        ...ownProps,
+        commandId,
+        commands: Object.values(getCommands(state)),
```

Comment:
Unless I'm mistaken, this will cause a re-render anytime something in the redux store changes since this will return a new array every time. It's not a big deal since not much should be changing in the redux store but it might be better to pass in the commands as a map and then convert to an array inside the component where needed. Other option is to add a selector that returns the array